src/initialization/initializeTypeScript/initializeTests.ts

```diff
@@ -4,6 +4,7 @@ const other = "other";
 
 export const initializeTests = async () => {
     const builtIn = await initializeBuiltInTests();
+    if (builtIn === "none") return;
```

Comment:
This `"none"` value is repeated twice. To keep a consistent constant, could you make a variable like `const none = "none";` above please?